test/torchaudio_unittest/transducer_test.py

```diff
@@ -0,0 +1,303 @@
+import numpy as np
+import torch
+
+from torchaudio_unittest import common_utils
+from torchaudio.prototype.transducer import RNNTLoss
+
+
+def get_basic_data(device):
+    # Test if example provided in README runs
+    # https://github.com/HawkAaron/warp-transducer
+
+    acts = torch.FloatTensor(
```

Comment:
nit: use the factory function ```torch.tensor([xyz], dtype=torch.float)``` instead of the type constructor. Same applies to IntTensor.